iib/workers/tasks/build.py

```diff
@@ -496,6 +497,65 @@ def _opm_index_add(
 
     with set_registry_token(overwrite_from_index_token, from_index):
         run_cmd(cmd, {'cwd': base_dir}, exc_msg='Failed to add the bundles to the index image')
+    
+    # add properties to bundles in the index
+    if add_properties:
+        maxOpenShiftVersionProperty = {
+            "type": "olm.maxOpenShiftVersion",
+            "value": "4.8",
+            "operatorbundle_version": "",
+            "operatorbundle-path": "",
+        }
+        for bundle in bundle_str.split(','):
+            if _sdk_is_valid(bundle):
```

Comment:
```suggestion
            if _requires_max_ocp_version(bundle):
```

Reply:
nit: I think would be better we add a comment and a name that clarifies better its intention.